src/Data/NodeResolver.php

```diff
@@ -201,8 +201,7 @@ public function resolve_uri( string $uri, $extra_query_vars = '' ) {
 					return null;
 				}
 
-				/** @todo resolve to an Archive Type. */
-				$post_type_object = get_post_type_object( $queried_object->post_type );
+				$post_type_object = get_post_type_object( 'post' );
```

Comment:
I'm not by my desk to check, but won't this break CPTs?

Reply:
@justlevine nope, this is within this condition `if ( $query->is_posts_page ) {` which we know is only for the `post` post type

Reply:
Okay phew!